### ISSUE TITLE
Regression: Help string lists `{OPTIONS}` after positional arguments

### ISSUE THREAD

**Issue by cbielow** [Taywee/args#111]
Using the current head version at 3e70c396bcdf2216846810988f967d88eb165782
I get
```
% Test.exe -h
  Test.exe [foo] [numbers...] {OPTIONS}
```
instead of the expected
```
% Test.exe -h
  Test.exe {OPTIONS} [foo] [numbers...] 
```

using code from the examples, i.e.
```
int main(int argc, char** argv)
{
  args::ArgumentParser parser("This is a test program.", "This goes after the options.");
  args::HelpFlag help(parser, "help", "Display this help menu", { 'h', "help" });
  args::ValueFlag<int> integer(parser, "integer", "The integer flag", { 'i' });
  args::ValueFlagList<char> characters(parser, "characters", "The character flag", { 'c' });
  args::Positional<std::string> foo(parser, "foo", "The foo position");
  args::PositionalList<double> numbers(parser, "numbers", "The numbers position list");
...
```

compiler: VS2022

I would assume the old behaviour is the correct one?

Great project btw :)

**Comment by Taywee** [comment]
This should be fixed as of release 6.4.5.  Please let me know if this doesn't solve it as expected.

**Comment by cbielow** [comment]
works. Thank you!
